libs/MeshKernelApi/include/MeshKernelApi/MeshKernel.hpp

```diff
@@ -632,6 +632,26 @@ namespace meshkernelapi
         /// @returns Error code
         MKERNEL_API int mkernel_get_geometry_error(int& invalidIndex, int& type);
 
+        /// @brief Get the integer indicating the interpolation type short
+        /// @param[out] type The integer indicating the interpolation type short
+        /// @returns Error code
+        MKERNEL_API int mkernel_get_interpolation_type_short(int& type);
```

Comment:
We should find a way around this for all enums... Too much boiler plate code to get a value.
I don't think it would be the end of the world to maintain enums in MKPy and MKNet every time an enum is changed in the back-end. We do that already in a few places.